### PR TITLE
allow for setting a configuration for date format

### PR DESCRIPTION
**Pull request by soberstadt**

### CODE REVIEW

File: app/assets/javascripts/fe/fe.common.js.erb [hunk 1]
```diff
@@ -57,12 +57,12 @@ function setUpSortables() {
...
 function setUpCalendars() {
-	now = new Date();
-	year = now.getFullYear() + 10;
-	$('[data-calendar]').datepicker({
+  now = new Date();
+  year = now.getFullYear() + 10;
+  $('[data-calendar]').datepicker({
     changeYear:true,
-		yearRange: '1950:' + year,
-    dateFormat: 'yy-mm-dd'
```
Comment (andrewroth) [inline app/assets/javascripts/fe/fe.common.js.erb]: Just wanted to check that you intended to change the default date format as well as per https://github.com/CruGlobal/fe/pull/63/files#diff-e54dd103df675ce9f0b934e94ff0ca8bR28 ?

Comment (soberstadt) [inline app/assets/javascripts/fe/fe.common.js.erb]: Good catch, my bad.

Comment (soberstadt) [inline app/assets/javascripts/fe/fe.common.js.erb]: Hows that?

**Comment by andrewroth** [comment]
Looks good from my perspective.  Not sure if I have authority to :+1: or if we need @twinge to look at it.

**Pull request merged**